packages/test/package.json

```diff
@@ -38,7 +38,7 @@
     "enzyme-adapter-react-16": "^1.15.2",
     "enzyme-to-json": "^3.4.3",
     "identity-obj-proxy": "^3.0.0",
-    "jest-cli": "^24.9.0",
+    "jest-cli": "^26.4.2",
```

Comment:
Did you figure out why we are using jest-cli instead of jest?

Reply:
`jest` seems to be a wrapper of `jest-cli` [see here](https://www.npmjs.com/package/jest), and `jest-cli` provides all the same packages (`jest/core` and `import-local`) that `jest` does, plus it's own.

I don't see any official documentation about the difference, maybe it exists, but I assume they are interchangeable.